### ISSUE TITLE
NuGet.VisualStudio.IVsPackageInstaller.InstallPackage should install latest version when no version given

### ISSUE THREAD

**Issue by jmevel** [NuGet/Home#7493]
## Details about Problem

As specified in the [NuGet.ClientsNuGet.VisualStudioExtensibilityIVsPackageInstaller.cs](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPackageInstaller.cs) class: 

```
/// <param name="version">
/// The version of the package to install. <c>null</c> can be provided to
/// install the latest version of the package.
/// </param>
```

NuGet product used: NuGet.VisualStudio package

Nuget.VisualStudio version 4.8.0

VS version: 15.8.9

OS version: Windows 10 version 10.0.17134

## Repro steps:

1.  Install an old version of a Nuget package on the target project

2. Execute the `IVsPackageInstaller.InstallPackage` method with a `null` package version

## Result
The latest version isn't installed

## Expected result
The latest version is installed and replace the current old package version

**Comment by jainaashish** [comment]
Currently it skips installing the package if the same id or version already installed. In your case, since you only provide the id so it only compares based on id and skip it. One workaround is to pass both id and new latest version then it will allow you to update to that version.

**Comment by jmevel** [comment]
Yes I implemented this workaround but `IVsPackageInstaller` doesn't have any method to get the last version of a package. 
So I had to use `NuGet.Client` to get an `IPackageSearchMetadata` and finally retrieve the latest version number. All this to finally install the latest version as expected.

I think the `InstallPackage` method should just do what it says no matter if an old version is already installed or not.
